docs/ref/databases.txt

```diff
@@ -542,23 +542,14 @@ Django doesn't provide an API to change them.
 .. _documented thoroughly: https://dev.mysql.com/doc/refman/en/charset.html
 
 By default, with a UTF-8 database, MySQL will use the
-``utf8_general_ci`` collation. This results in all string equality
+``utf8mb4_0900_ai_ci`` collation. This results in all string equality
 comparisons being done in a *case-insensitive* manner. That is, ``"Fred"`` and
 ``"freD"`` are considered equal at the database level. If you have a unique
 constraint on a field, it would be illegal to try to insert both ``"aa"`` and
 ``"AA"`` into the same column, since they compare as equal (and, hence,
 non-unique) with the default collation. If you want case-sensitive comparisons
 on a particular column or table, change the column or table to use the
-``utf8_bin`` collation.
-
-Please note that according to `MySQL Unicode Character Sets`_, comparisons for
-the ``utf8_general_ci`` collation are faster, but slightly less correct, than
-comparisons for ``utf8_unicode_ci``. If this is acceptable for your application,
-you should use ``utf8_general_ci`` because it is faster. If this is not acceptable
-(for example, if you require German dictionary order), use ``utf8_unicode_ci``
-because it is more accurate.
```

Comment:
I'm not sure why the paragraph is deleted, I think this is still true but we should update to say ` utf8mb4_general_ci` and `utf8mb4_unicode_ci` 
See the "_general_ci Versus _unicode_ci Collations" section in: https://dev.mysql.com/doc/refman/en/charset-unicode-sets.html

Reply:
Some users might wonder why we go from talking about `utf8mb4_0900_ai_ci` and `utf8mb4_0900_as_cs` in the paragraph above, to these other collations here. But I'm fine with however you want to handle it.